plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -4,7 +4,9 @@ import java.io.OutputStreamWriter
 import java.nio.file.Path
 
 import sbt.Keys._
-import sbt._
+// format: off
```

Comment:
why is this necessary? if this is scala3 syntax, better to modify scalafmt configuration, no?

Reply:
https://github.com/scalameta/sbt-scalafmt/blob/0344d19d9d6e9c8d8944d9408f7774c79046c332/.scalafmt.conf#L23-L24

without this comment expand import

```diff
- import sbt.{given, _}
+ import sbt._
+ import sbt.given
```

but error if Scala 2.12

- https://github.com/scala/scala/pull/9736
- https://github.com/sbt/sbt/issues/7737

Reply:
is this is an unreported bug in scalafmt?